src/commands/redteam.ts

```diff
@@ -126,9 +170,14 @@ export function redteamCommand(program: Command) {
 
       // Create config file
       const config = {
-        prompts: [prompt],
+        prompts,
         providers: [provider],
         tests: [],
+        redteam: redteamConfigSchema.safeParse({
+          plugins: plugins,
+          strategies: strategies,
+          numTests,
+        }).data,
```

Comment:
Because we only ask the user to set `numTests` globally, it would be much nicer to generate:

```yaml
redteam:
  numTests: 5
  plugins:
    - competitors
    - contracts
    - debug-access
    - ...
```
this would make it a lot cleaner and more approachable as part of onboarding IMO